lib/Doctrine/deploy/sampleData/ServiceEndpoints.xml

```diff
@@ -53,15 +53,15 @@
         <CORE></CORE>
         <IN_PRODUCTION>N</IN_PRODUCTION>
         <NODE_MONITORED>N</NODE_MONITORED>
-        <SITENAME>SWITCH</SITENAME>
+	<SITENAME>Torch</SITENAME>
         <COUNTRY_NAME>Switzerland</COUNTRY_NAME>
         <COUNTRY_CODE>CH</COUNTRY_CODE>
         <ROC_NAME>NGI_CH</ROC_NAME>
         <URL></URL>
     </SERVICE_ENDPOINT>
     <SERVICE_ENDPOINT PRIMARY_KEY="43873G0">
         <PRIMARY_KEY>43873G0</PRIMARY_KEY>
-        <HOSTNAME>bdii.lhep.unibe.ch</HOSTNAME>
+        <HOSTNAME>bdii.lhep.unilu.grid.ch</HOSTNAME>
```

Comment:
Hmmmm `unilu.ch` is a thing, maybe we could change `unilu.grid.ch` to something else - perhaps `torch.grid.ch`?

Reply:
or perhaps something different so we can change the `unilu` sitename to something different to `torch`.

Reply:
ok, I'll change the sitename and hence its' services